packages/confirm-dialog/test/confirm-dialog.test.js

```diff
@@ -549,6 +549,10 @@ describe('vaadin-confirm-dialog', () => {
         await oneEvent(overlay, 'vaadin-overlay-open');
       });
 
+      afterEach(() => {
+        confirm.opened = false;
+      });
```

Comment:
Needed to add this to explicitly close confirm dialogs before opening new ones. Otherwise there would be multiple dialogs opened at the same time, which would add a `z-index` to the `style` attribute tested below, causing tests to fail.